torchaudio/csrc/rnnt/cpu/cpu_transducer.h

```diff
@@ -36,15 +36,15 @@ status_t Compute(
   const int& D = options.numTargets_;
 
   { // compute denominators.
-    status_t status = LogSumExp2D<DTYPE, CAST_DTYPE>(
```

Comment:
let's mark as follow-up for later to make sure to check the status of the output, and whether any error followed